src/cdk/v2/destinations/bloomreach/procWorkflow.yaml

```diff
@@ -98,22 +120,50 @@ steps:
         "event_type": $.context.event,
       });
 
-      $.context.payload = $.removeUndefinedAndNullValues({name: $.CUSTOMER_EVENT_COMMAND, data})
+      $.context.payload = $.removeUndefinedAndNullValues({name: $.CUSTOMER_EVENT_COMMAND, data});
+      $.context.endpoint = $.getBatchEndpoint(.destination.Config.apiBaseUrl, .destination.Config.projectToken);
+      $.context.method = 'POST';
+
+  - name: prepareRecordPayload
+    condition: $.context.messageType === {{$.EventType.RECORD}}
+    steps:
+      - name: prepareInsertActionPayload
+        condition: .message.action === {{$.RecordAction.INSERT}}
+        template: |
+          const fields = .message.fields;
+          const item_id = fields.item_id;
+          fields.item_id = null;
+          const properties = $.removeUndefinedAndNullValues(fields);
```

Comment:
Instead of repeating this 3 times, we can move them in common place.
Also use this
const fields = .message.fields;
const item_id = fields.item_id;
delete fields.item_id;

Reply:
we are using these for insert and update. And for delete, payload is different. I kept all these in separate steps, due to code readability and for debugging the future issues easily.

Should I move these to other place?